base/main_menus/InnMenu.ts

```diff
@@ -0,0 +1,197 @@
+import {YesNoMenu} from "../windows/YesNoMenu";
+import {DialogManager} from "../utils/DialogManager";
+import {GoldenSun} from "../GoldenSun";
+import {TextObj, Window} from "../Window";
+import * as _ from "lodash";
+import {Graphics} from "phaser-ce";
+
+export class InnMenu {
+    public game: Phaser.Game;
+    public data: GoldenSun;
+    public close_callback: Function;
+
+    public choice_menu: YesNoMenu;
+    public dialog_manager: DialogManager;
+    public coin_window: Window;
+    public coin_number: TextObj;
+    public callback: Function;
+    public mask: Graphics;
+    public circle: Graphics;
+
+    public inn_id: string;
+    public avatar: string;
+    public cost: number;
+    public message: string;
+
+    constructor(game: Phaser.Game, data: GoldenSun) {
+        this.game = game;
+        this.data = data;
+
+        this.dialog_manager = new DialogManager(this.game, this.data);
+        this.choice_menu = new YesNoMenu(this.game, this.data);
+        this.coin_window = new Window(this.game, 0, 128, 96, 28);
```

Comment:
Use these dimensions to get a similar window to the original game:

![image](https://user-images.githubusercontent.com/7890647/103393594-30762900-4b02-11eb-8358-9a1d792814a8.png)